llpc/util/llpcElfWriter.cpp

```diff
@@ -836,6 +889,28 @@ void ElfWriter<Elf>::UpdateMetaNote(
     pNewNote->pData = pData;
 }
 
+// Retrieves the section data for the specified section name, if it exists.
+template<class Elf>
+Result ElfWriter<Elf>::GetSectionData(
+        const char*  pName,       // [in] Name of the section to look for
+        const void** pData,       // [out] Pointer to section data
+        size_t*      pDataLength  // [out] Size of the section data
+) const
+{
+    Result result = Result::ErrorInvalidValue;
+
+    auto pEntry = m_map.find(pName);
+
+    if (pEntry != m_map.end())
```

Comment:
Prefer to remove the line of whitespace here, due to the tight logical connection between the find and the if-block.

Reply:
Done.